detox/src/devices/drivers/android/emulator/EmulatorDriver.js

```diff
@@ -107,6 +102,11 @@ class EmulatorDriver extends AndroidDriver {
     });
```

Comment:
Regarding your `_waitForBootToComplete` implementation — how well does it play with Jest timeouts? I have a bad gut feeling here, honestly, @d4vidi . ⚠️

Reply:
Where do you handle a cleanup call, which cancels the `_waitForBootToComplete` routine, Amit?